scripts/cfg_files.py

```diff
@@ -43,19 +43,25 @@ def write_cfg_files(cfg: config.Config, lib_spec: config.LibrarySpec) -> None:
 
     # No semihosting and no linker script
     nosys_lines = base_cfg_lines + [
-        '$@/../lib/clang-runtimes/{}/lib/crt0.o'.format(target),
+        '$@/../lib/clang-runtimes/{}/lib/crt0.o'.format(lib_spec.name),
+        '-L$@/../lib/clang-runtimes/{}/lib'.format(lib_spec.name),
```

Comment:
This line and the next one are common between all configs, so they should be moved to `base_cfg_lines`